docs/en/latest/USER_GUIDE.md

```diff
@@ -23,7 +23,7 @@ title: User Guide
 
 The following are parts of the modules' snapshot.
 
-## Dashboard
+## Dashboard {#apisix-dashboard}
```

Comment:
Could you please add a screenshot on this first?

Reply:
yes sure

Reply:
<img width="463" alt="image" src="https://user-images.githubusercontent.com/40708551/123239877-6c140d80-d4fd-11eb-938b-fa0ce5bd4e7e.png">

<img width="912" alt="image" src="https://user-images.githubusercontent.com/40708551/123240010-8e0d9000-d4fd-11eb-966b-69a6f599df5d.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/40708551/123240310-d462ef00-d4fd-11eb-8235-d864a1e063ab.png">

Reply:
see id changes

Reply:
ok, got it